src/index.js

```diff
@@ -130,9 +132,10 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
           events = [events];
         }
         events.forEach(eventName => {
-          const handlerOptions = !this.props.preventDefault && ['touchstart', 'touchmove'].indexOf(eventName) !== -1
-            ? { passive: true }
-            : null;
+          const handlerOptions =
```

Comment:
Yes, all you have said is true - if we follow the standard and all. However I suspect that PhantomJS (as its implementing the standard) introduced a bug and it stores all arguments references used in the `addEventListener` call to compare them with equality check later when executing `removeEventListener` call.

I've filed an issue on the PhantomJS for this, but no answer - https://github.com/ariya/phantomjs/issues/14982